### ISSUE TITLE
I have issues with Editor text box.

### ISSUE THREAD

**Issue by roopeshkurian** [ghiscoding/Angular-Slickgrid#1303]
### Describe the bug

In the text editor cursor cannot be moved. But if we use arrow keys from the keyboard we can move the cursor. The entire text is always being selected when we use the mouse. 

### Reproduction

I am referring to your online example 
https://ghiscoding.github.io/Angular-Slickgrid/#/editor

I am trying to edit City of Orgin.
When I double-click on "Boston, MA, United States" the entire text is being selected.  I am not able to move the cursor between the letters or words. But if I use arrow keys on the keyboard the cursor can be moved.

### Expectation

In the editor text box, I should be able to move the cursor and make edits to the text. 

### Environment Info

```shell
Angular 16.2.12
angular-slickgrid 6.4.0
TypeScript 5.1.6
Browser Mozilla Firefox.
System Windows 11
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/Angular-Slickgrid/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [HOWTO - Step by Step](https://github.com/ghiscoding/angular-slickgrid/wiki/HOWTO---Step-by-Step).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/Angular-Slickgrid/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/Angular-Slickgrid/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by roopeshkurian** [comment]
Could you kindly watch the video provided in the link below ?

https://watch.screencastify.com/v/oWQyhExOUiYPa1ReVa0i

I could use a mouse click and insert content in between. Or I can select a particular word and then delete only that word. 

Can you please check the video below   

https://watch.screencastify.com/v/rrkn1ZS4o5Usv64lMiyw

In your example, I cannot select a word or focus between words.

I was able to accomplish it by using the composite editor.

https://ghiscoding.github.io/Angular-Slickgrid/#/composite-editor

I will  try to use it .. 

Thank you so much for your help.
Sorry for the trouble

**Comment by ghiscoding** [comment]
this was hard to troubleshoot, I think I found the problem in the `SlickCellRangeSelector` which was cancelling the event bubbling and by doing so was breaking some behavior like clicking inside the editor. I have a fix in Slickgrid-Universal [PR](https://github.com/ghiscoding/slickgrid-universal/pull/1183)

**Comment by roopeshkurian** [comment]
I am glad that we found the issue. Thank you so much for solving it.

**Comment by ghiscoding** [comment]
This is fixed now and available in [v6.5.0](https://github.com/ghiscoding/Angular-Slickgrid/releases/tag/v6.5.0), you can give a ⭐ if you like the project. Thanks for the feedback

**Comment by roopeshkurian** [comment]
Thank you very much for your wonderful project. I truly appreciate it. I have already rated it with  ⭐
